src/main/java/athena/logic/Parser.java

```diff
@@ -55,16 +58,20 @@ public static String getParameterDesc(String taskInformation, String delimiter,
             param = defaultValue;
         } else {
             String[] retrieveParamInfo = taskInformation.split(delimiter, 2);
-            String retrievedParamInfo = retrieveParamInfo[1];
-            int paramNextSlash = retrievedParamInfo.indexOf("/");
-            if (paramNextSlash == -1) {
-                param = retrievedParamInfo;
-            } else {
+            CharSequence retrievedParamInfo = retrieveParamInfo[1];
+
+            String patternStr = "\\s[a-zA-Z]\\/";
```

Comment:
Change to `"\\s\\w+\\/"` to fix #199.

Now it only detects one alphabet before the `/`. `\\s\\w+\\/` will make it detect any number of alphanumeric characters before the `/`, in case the user provides an invalid parameter like `abc/`.